### ISSUE TITLE
Encryption key is appended with 'passpharase'

### ISSUE THREAD

**Issue by usma0118** [tiredofit/docker-db-backup#362]
Encryption key is appended with '${mypasspharase}'
from debug log i can see

```
echo ''\''mypasspharase'\'''
this results in 'mypasspharase'

Encrypting with GPG Passphrase
++ date +%s
+ encrypt_routines_start_time=1723535468
++ run_as_user mktemp -d
++ sudo -Eu dbbackup mktemp -d
+ encrypt_tmp_dir=/tmp/tmp.QapNZ1RJNl
**+ echo ''\''***4.QAFe4-*******A-***!*******************************-X'\'''**
**output: + '***4.QAFe4-*******A-***!*******************************-X'**
+ silent run_as_user gpg --batch --home /tmp/tmp.QapNZ1RJNl --yes --passphrase-fd 0 -c /data/01_dbbackup.6nF7Ax/mariadb_mariadb.photoview.svc.cluster.local_20240813-095107.sql.zst
+ '[' true = true ']'
```
**+ echo ''\''***4.QAFe4-*******A-***!*******************************-X'\'''**
**output: + '***4.QAFe4-*******A-***!*******************************-X'**

if i add '' to my passpharase than extraction works as expected.

**Comment by tiredofit** [comment]
Quotes! Can you try this test image to see if this will resolve the problem:

`docker.io/tiredofit/db-backup:develop-encrypt_quotes`

**Comment by usma0118** [comment]
Unfortunately no. tested. Quotes are still included.

**Comment by tiredofit** [comment]
Thanks. Apologies on the wild testing. I have a feeling this is something related to when I built the routines for multiple db backups. Will be back here in ~15 minutes to confirm and see if I can resolve in short order.

**Comment by tiredofit** [comment]
Found it. Would you mind pulling that same image again and seeing if this resolves. This occurred with a few other variables in the 4.0.x release where quotes were added unnecessarily when auto generating multi db backup routines.

**Comment by usma0118** [comment]
Works as expected. it's fixed. Thanks for quick resolution.

**Comment by tiredofit** [comment]
Thanks for the initial reconnaissance :) It's been tagged and released as `4.1.4` and a new image should be appearing in the usual places momentarily.
